mws/future_utils/__init__.py

```diff
@@ -0,0 +1,39 @@
+from .collections import unique_list_order_preserved
```

Comment:
could there end up with a namespace conflict here, collections is a python standard library?

I've had weird stuff like this happen before, it shouldn't happen, but sometimes does!

Reply:
That is possible, though I think it's less an issue in Python 3 with the absolute imports.

If we're still trying to support Py2 for compatibility in 0.8, then yes that could be a minor issue. Might have to pepper in `from __future__ import absolute_import` to be safe there.

Reply:
> That is possible, though I think it's less an issue in Python 3 with the absolute imports.
> 
> If we're still trying to support Py2 for compatibility in 0.8, then yes that could be a minor issue. Might have to pepper in `from __future__ import absolute_import` to be safe there.

hmmmmm we've pretty much ditched py2 haven't we so don't put more work in than is necessary.